tests/conftest.py

```diff
@@ -0,0 +1,12 @@
+"""
+Global pytest config, fixtures, and helpers go here!
+"""
+
+# Standard
+import os
+import sys
+
+# Make sure tests can import torchchat
```

Comment:
This would be a lot cleaner if we move to having a `pyproject.toml` or `setup.py` to bundle `torchchat` as a package that could be installed with `pip install -e`.

Reply:
on the list